vortex-fastlanes/src/for/compute.rs

```diff
@@ -86,4 +94,31 @@ mod test {
         assert_eq!(scalar_at(&forarr, 1).unwrap(), 15.into());
         assert_eq!(scalar_at(&forarr, 2).unwrap(), 19.into());
     }
+
+    #[test]
+    fn for_as_contiguous() {
```

Comment:
i guess we should probably centralize 1-2 smoketests for each of the `ArrayCompute` functions for all of the variants.